interfaces/cython/cantera/ck2cti.py

```diff
@@ -65,6 +65,8 @@
                 'KJOULES/MOL': 'kJ/mol',
                 'KJOULES/MOLE': 'kJ/mol'}
 
+Avogadro = 6.02214129e23  # in molec/mol
```

Comment:
Can you document where this number comes from? For instance, it is different from the CODATA value: https://physics.nist.gov/cgi-bin/cuu/Value?na

Reply:
It is the 2010 CODATA value, consistent with the value in `ct_defs.h`.